docs/editor/setup.md

```diff
@@ -29,13 +29,22 @@ Now, you can simply type `code .` in any folder to start editing files in that f
 2. Make a new folder and extract `VSCode-linux-x64.zip` inside that folder.
 3. Double click on `Code` to run Visual Studio Code.
 
->**Tip:** If you want to run VS Code from the terminal, create the following link substituting `/path/to/vscode/Code` with the absolute path to the `Code` executable
+>**Tip:** If you want to run VS Code from the terminal, run the following to setup a script in `/usr/local/bin` to run Visual Studio Code in the background and hide its output. Replace `/path/to/vscode/Code` with the actual path to the executable.
 
 ```
-sudo ln -s /path/to/vscode/Code /usr/local/bin/code
+VSCODE_PATH=/path/to/vscode/Code
+echo 'nohup '$VSCODE_PATH' "$@" > ~/.vscode/nohup.out 2>&1 &' | sudo tee /usr/local/bin/code > /dev/null

```

Comment:
Hey @hashnar, there will likely be a deb package as part of the next release. Auto updating and rpm package to follow. 

This part of the documentation will be removed soon as a result.